tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -777,8 +694,7 @@ module.exports = {
       locateStrategy: 'xpath'
     },
     fileLinkInFileRow: {
-      selector: '//span[contains(@class, "file-row-name")]',
-      locateStrategy: 'xpath'
```

Comment:
why deleting that locateStrategy?

Reply:
Because is used by `getFileLinkSelectorByFileName` so thought it would be better in filesList PO